### PR TITLE
SPFx is not supposed to be used for branding

### PR DESCRIPTION
**Pull request by westleyMS**
The only thing supported by SPFx regarding branding is header/footer customizations.
If you try to change theme elements, you will have a flashing experience at the start of the page load where you see the OOB page, and then the SPFx stuff is added. This is not avoidable and is why we have no support for it. You might want to add more about the supported ways, here are some related links-
https://docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/site-theming/sharepoint-site-theming-overview
https://docs.microsoft.com/en-us/sharepoint/dev/transform/modernize-branding

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/branding-sharepoint-online-sites-modern-experience.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Below is a summary of the differences between branding a site in the classic exp
 | Classic experience | Modern experience |
 |:-----|:-----|
 |Unresponsive, OOB themes that you can customize|Responsive themes that look great on any device and can be customized to match your brand|
-|Use custom master pages and CSS that are ignored in the modern experience|Use the [SharePoint Framework](/sharepoint/dev/spfx/sharepoint-framework-overview) to add header and footer, customize theme colors, etc.|
+|Use custom master pages and CSS that are ignored in the modern experience|Use the [SharePoint Framework](/sharepoint/dev/spfx/sharepoint-framework-overview) to add header and footer. Site templates and tenant settings offer some custom options|
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/branding-sharepoint-online-sites-modern-experience.md:58]:
```suggestion
|Use custom master pages and CSS that are ignored in the modern experience|Use the [SharePoint Framework](/sharepoint/dev/spfx/sharepoint-framework-overview) to add headers and footers. Site templates and tenant settings offer some custom options|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @westleyMS, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by MikePlumleyMSFT** [comment]
@MattEEvans can you review this PR?

**Pull request merged**